docker/Dockerfile.armv7-linux-androideabi

```diff
@@ -56,11 +59,16 @@ ENV CARGO_TARGET_ARMV7_LINUX_ANDROIDEABI_LINKER="$CROSS_TOOLCHAIN_PREFIX"gcc \
     SIZE_armv7_linux_androideabi="$CROSS_TOOLCHAIN_PREFIX"size \
     STRINGS_armv7_linux_androideabi="$CROSS_TOOLCHAIN_PREFIX"strings \
     STRIP_armv7_linux_androideabi="$CROSS_TOOLCHAIN_PREFIX"strip \
+    CMAKE_TOOLCHAIN_FILE_armv7_linux_androideabi=/opt/toolchain.cmake \
     BINDGEN_EXTRA_CLANG_ARGS_armv7_linux_androideabi="--sysroot=$CROSS_SYSROOT" \
     DEP_Z_INCLUDE="$CROSS_SYSROOT/usr/include/" \
     RUST_TEST_THREADS=1 \
     HOME=/tmp/ \
     TMPDIR=/tmp/ \
     ANDROID_DATA=/ \
     ANDROID_DNS_MODE=local \
-    ANDROID_ROOT=/system
+    ANDROID_ROOT=/system \
+    CROSS_CMAKE_SYSTEM_NAME=Android \
+    CROSS_CMAKE_SYSTEM_PROCESSOR=armv7-a \
+    CROSS_CMAKE_CRT=android \
+    CROSS_CMAKE_OBJECT_FLAGS="--target=arm-linux-androideabi -DANDROID -ffunction-sections -fdata-sections -fPIC --target=armv7-linux-androideabi"
```

Comment:
I believe this change is correct, with `--target` provided twice, since that's what `cmake-rs` does as of `0.1.49` and `cc-rs` as of `1.0.77` (this is using Rust 1.64.0).